Modules/_io/stringio.c

```diff
@@ -463,22 +463,24 @@ _io_StringIO_truncate_impl(stringio *self, Py_ssize_t size)
 
 /*[clinic input]
 _io.StringIO.seek
-    pos: Py_ssize_t
-    whence: int = 0
+    offset as pos: Py_ssize_t
+    whence: int(c_default='0') = io.SEEK_SET
     /
 
-Change stream position.
+Change the stream position and return the new absolute position.
 
-Seek to character offset pos relative to position indicated by whence:
-    0  Start of stream (the default).  pos should be >= 0;
-    1  Current position - pos must be 0;
-    2  End of stream - pos must be 0.
-Returns the new absolute position.
+Offsets relative to the start of the stream are usually zero or positive.
+Offsets relative to the current and end of file positions must zero;
+any other values are unsupported, and will raise OSError.
+
+Some platforms allow seeking beyond the end of a file.
```

Comment:
Is the behavior of StringIO platform-depending?

Reply:
No, this is a copy and paste error. Thanks for noticing.